functions/src/test/java/org/apache/geode/examples/functions/ExampleTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.examples.functions;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.geode.cache.Region;
+import org.apache.geode.cache.execute.Execution;
+
+import org.geode.examples.util.Mocks;
+import org.junit.Test;
+
+public class ExampleTest {
+
+  @Test
+  public void testExample() throws Exception {
+    Example example = new Example(10);
+
+    Region<Integer, String> region = Mocks.region("example-region");
+    List<Integer> result = Arrays.asList(1, 2, 3, 5, 7);
+    Execution execution = Mocks.execution(PrimeNumber.ID, result);
+
+    example.accept(region, execution);
+    System.err.println("size=" + example.getPrimes().size());
+    assertEquals(5, example.getPrimes().size());
```

Comment:
Wouldn't it be better to assert that result .equals primes?

Reply:
Good note. Wilco.